moler/config/loggers.py

```diff
@@ -90,6 +95,46 @@ def _get_moler_version_cloned_from_git_repository(setup_py_path):
     return "{} cloned from git repository".format(version)
 
 
+def set_backup_count(backup_count):
+    """
+    Set interval for rotating logging. If parameter is not an int number then the function does not change any value.
```

Comment:
It seems that docs is from set_interval method :)

Reply:
Yes. Ctr+c, ctr+v issue :)